### PR TITLE
Trend micro vision one

### PR DESCRIPTION
**Pull request by shaqnawe**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added outputs to get endpoint info and get alert details.

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Review by amshamah419 — COMMENTED** [review]
Thanks for your contribution @shaqnawe! Just a few comments, but otherwise it looks pretty straightforward. Can you add release notes to this PR? You can do so by running `demisto-sdk update-release-notes -i Packs/TrendMicroVisionOne`. This will do two things: Bump the pack version, and also generate a release note file. In the file, just write down what the changes are (we can help with the wording).

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml [hunk 1]
```diff
@@ -1372,7 +1431,7 @@ script:
       type: string
     description: Displays a list of Observed Attack Techniques events that match the specified criteria.
     name: trendmicro-visionone-get-observed-attack-techniques
-  dockerimage: demisto/pytmv1:0.8.7.104820
+  dockerimage: devtesting/pytmv1:0.9.0.1730248273
```
[the remark this answers is not in the document]
Comment (samuelFain) [reply Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml:1434]: @amshamah419 FYI, this this is matching dockerfiles PR: https://github.com/demisto/dockerfiles/pull/34200.

[the remark this answers is not in the document]
Comment (shaqnawe) [reply Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml:1434]: I have closed the aforementioned dockerfiles PR and created a new one https://github.com/demisto/dockerfiles/pull/34362.

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3_test.py [hunk 2]
```diff
@@ -1567,10 +1567,12 @@ def test_delete_custom_script(mocker):
...
 def add_custom_script_mock_response(*args, **kwargs):
     with open("./test_data/add_custom_script.json") as f:
-        return_value: dict[str, str] = json.load(f)
+        json.load(f)
     return Result(
         result_code=ResultCode.SUCCESS,
-        response=AddCustomScriptResp(**return_value),
+        response=AddCustomScriptResp(
+            **{"scriptId": "44c99cb0-8c5f-4182-af55-62135dbe32f1"}
+        ),
```
[the remark this answers is not in the document]
Comment (shaqnawe) [reply Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3_test.py:1575]: This was implemented to circumvent and issue but it has now been resolved, I have reverted the change.

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (amshamah419) [reply Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml]: Hi @shaqnawe, I reviewed the dockerfiles pr and approved it. It looks like a required job isn't getting triggered so I will ping @kobymeir to take a quick look.

**Comment by amshamah419** [comment]
Once we get the dockerfiles pr merged, the pre-commit should run. Otherwise I think the PR is in good shape and good to merge.

**Review by amshamah419 — APPROVED** [review]
Looks great!

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (amshamah419) [reply Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml]: Hi @shaqnawe I ran the testing on gitlab and it looks like the docker image is good to go. I approved and merged the PR so we can get a new docker tag for you. I'll edit this message when it's created.

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml [hunk 3]
```diff
@@ -1372,7 +1431,7 @@ script:
       type: string
     description: Displays a list of Observed Attack Techniques events that match the specified criteria.
     name: trendmicro-visionone-get-observed-attack-techniques
-  dockerimage: demisto/pytmv1:0.8.7.104820
+  dockerimage: devtesting/pytmv1:0.9.1.1732053127
```
[the remark this answers is not in the document]
Comment (shaqnawe) [reply Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml]: Done!

File: Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml [hunk 4]
```diff
@@ -359,6 +359,21 @@ script:
...
+      type: string
+    - contextPath: VisionOne.Endpoint_Info.componentVersion
+      description: The agent component version.
+      type: string
+    - contextPath: VisionOne.Endpoint_Info.policyName
+      description: The name of a policy for an event.
+      type: string
+    - contextPath: VisionOne.Endpoint_Info.protectionManager
+      description: The name of your protection manager.
+      type: string
```
[the remark this answers is not in the document]
Comment (shaqnawe) [reply Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml:376]: Hey @amshamah419 It's supposed to be snake case, I'm certain I had updated it on my side.

**Pull request merged**